src/vs/gitpod/browser/workbench/workbench.html

```diff
@@ -3,6 +3,30 @@
 <html>
 	<head>
 		<script>
+
+			window.gitpodIDEErrors = []
+
+			const onVsCodeWorkbenchError = (event) => {
+				const flags = new Map();
+
+				// We take a look at what is the resource that was attempted to load;
+				const resourceSource = event?.target?.getAttribute('src') || event?.target?.getAttribute('href');
```

Comment:
`?.` is safe to use in all browsers already? this code is not transpiled

Reply:
Good point, I'm not sure whether we support IE in Gitpod...
<img width="836" alt="image" src="https://user-images.githubusercontent.com/29888641/185582095-1fee2346-afb3-4739-9307-23cc2deb3b21.png">

Reply:
https://caniuse.com/mdn-javascript_operators_optional_chaining

Reply:
we don't support IE, if all other support let's use it.